src/code_80027D00.c

```diff
@@ -4675,6 +4679,8 @@ GLOBAL_ASM("asm/non_matchings/code_80027D00/func_80030C34.s")
 
 #ifdef MIPS_TO_C
 //generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
+// This is likely the function responsible for detecting triple-tap "A"
+// for "fast" acceleration
```

Comment:
Is it intentional that you marked two functions with this comment? If so, it's fine. Just checking.

Reply:
Yes, although I should have made that clearer. It looks like there are 2 functions that each touch D_80165460 and while I haven't looked at them too carefully they both seem to do the same thing. So there might be slight differences, or maybe they're actually copies of each other.